src/ensembl/src/content/app/custom-download/containers/content/result-holder/resultHolderHelper.ts

```diff
@@ -1,81 +1,96 @@
 import config from 'config';
 
-export const getEndpointUrl = (attributes: any) => {
-  let endpoint = config.genesearchAPIEndpoint + '/genes/query?query=';
-
-  let endpointFields = '';
-  attributes.forEach((attribute: any) => {
-    endpointFields += attribute[2] + ',';
+import { keys, mapKeys, set, get, trim } from 'lodash';
+import JSONValue from 'src/shared/types/JSON';
+
+export const getProcessedAttributes = (flatSelectedAttributes: JSONValue) => {
+  const filteredAttributes = keys(flatSelectedAttributes).filter(
+    (key) => flatSelectedAttributes[key]
+  );
+  return filteredAttributes.map((value: string) => {
+    return value
+      .split('.default.')
+      .join('.')
+      .split('genes.')
+      .join('');
```

Comment:
is this the same as `value = value.replace('.default.', '.'); value = value.replace('genes.', '')`?

Reply:
`.replace` only replaces the first match. This string could have multiple '.default.'.

Reply:
@msmanoj You can replace all with `.replace` if you use regex with the global flag. For example: `str.replace(/\.default/g, '.').replace(/genes\./g, '')`.

Reply:
That's done